pkgs/data/fonts/iosevka/comfy.nix

```diff
@@ -23,25 +23,40 @@ let
     sha256 = "1h72my1s9pvxww6yijrvhy7hj9dspnshya60i60p1wlzr6d18v3p";
   };
   privateBuildPlan = src.outPath + "/private-build-plans.toml";
-  overrideAttrs = (attrs: {
-    inherit version;
-
-    meta = with lib; {
-      inherit (src.meta) homepage;
-      description = ''
-        Customised build of the Iosevka typeface, with a consistent
-        rounded style and overrides for almost all individual glyphs
-        in both roman (upright) and italic (slanted) variants.
-      '';
-      license = licenses.ofl;
-      platforms = attrs.meta.platforms;
-      maintainers = [ maintainers.DamienCassou ];
-    };
-  });
   makeIosevkaFont = set:
-    (callPackage ./. { inherit set privateBuildPlan; }).overrideAttrs
-    overrideAttrs;
-in builtins.listToAttrs (builtins.map (set: {
-  name = set;
-  value = makeIosevkaFont set;
-}) sets)
+    let superBuildNpmPackage = buildNpmPackage; in
+    (iosevka.override rec {
+      inherit set privateBuildPlan;
+      buildNpmPackage = args: superBuildNpmPackage
+        (args // {
+          inherit version;
+
+          src = fetchFromGitHub {
+            owner = "be5invis";
+            repo = "iosevka";
+            rev = "ad1e247a3fb8d2e2561122e8e57dcdc86a23df77";
```

Comment:
why this revision and not a tag?

Reply:
Comfy's README specifies the revision it was built against and the release. The revision is the more specific of the two in my mind.

Reply:
ok for me